### ISSUE TITLE
Updating contacts date when the SF tab is clicked

### ISSUE THREAD

**Issue by petednz** [progressivetech/net.ourpowerbase.sumfields#35]
I just noticed that clicking the Summary Fields tab for a contact wasn't loading the latest Contributions that have been added. Is this something you would accept a PR for?

**Comment by petednz** [comment]
Ie. I go to a contact and add a contribution, then click on Summary Fields but latest doesn't show. If i reload the Contact then it shows.

**Comment by jmcclelland** [comment]
Yes - that would be great. Not sure the best strategy for it, but if you have ideas I am all for it.
